### PR TITLE
 adds comment fot IT at home

### LINKED ISSUE #3757: Please add a hidden string to the landing page for monitoring purpose
### Describe the problem and steps to reproduce it:
The old desktop site has a hidden string on the landing page for Ops to do site availability monitoring, for example https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/templates/addons/home.html#L33-L34 

### What happened?
The new mobile site doesn't have such a string. We're using "Firefox" for now.
### What did you expect to happen?
Can you add a hidden string on the landing page so we can use to monitor the site availability? Ideally, please put a note that says "not remove it" so it won't get removed by accident.

### Anything else we should know?
We have no preference what the hidden string is, as long as it's unique and not too long. :)

### PR DESCRIPTION
**Pull request by tsl143**
Fixes #3757

### CODE REVIEW

File: src/amo/components/Home/index.js [hunk 1]
```diff
@@ -170,6 +170,9 @@ export class HomeBase extends React.Component {
 
     return (
       <div className="Home">
+        {/* IT looks for this comment with nagios, don't remove it. */}
+        <span dangerouslySetInnerHTML={{ __html: '<!-- aww yeah -->' }} />
```
Comment (tofumatt) [inline src/amo/components/Home/index.js]: So this should probably just be an HTML comment–won't that work? Or will react strip it out?

Also I wish we used a more fun string. This is what IT look for on addons-server so maybe ours should be different.

Maybe we could use "Godzilla of browsers" or something cute 😉

Comment (willdurand) [reply src/amo/components/Home/index.js]: > Or will react strip it out?

I am pretty sure React removes all comments.

Comment (tofumatt) [reply src/amo/components/Home/index.js]: Ah okay I figured that might be the case. Cool then!

Comment (tsl143) [reply src/amo/components/Home/index.js]: Yes react removes all the comments.
And yes, writing test case is much better idea.
`Godzilla of browsers` \o/

File: tests/unit/amo/containers/TestHome.js [hunk 2]
```diff
@@ -168,6 +168,11 @@ describe(__filename, () => {
     });
   });
 
+  it('renders a span with comment for IT', () => {
+    const root = render();
+    expect(root.find('[data-comment="For-IT"]')).toHaveLength(1);
```
Comment (willdurand) [inline tests/unit/amo/containers/TestHome.js]: I'd get rid of the `data-` attribute and I'd try to assert `root.html()` to be sure to find the expected HTML comment.

Comment (tsl143) [reply tests/unit/amo/containers/TestHome.js]: @willdurand , I got the `root.html()` test but I am not sure what else to use if not `data-comment`

Comment (willdurand) [reply tests/unit/amo/containers/TestHome.js]: nothing. why do you need this?

Comment (tsl143) [reply tests/unit/amo/containers/TestHome.js]: How to search for element with specific text, either to put class or a data-attribute?

Comment (willdurand) [reply tests/unit/amo/containers/TestHome.js]: You don't. Can't you use `toContain()`?

Comment (tsl143) [reply tests/unit/amo/containers/TestHome.js]: I tried to use this
`  it('renders a span with comment for IT', () => {`
`   const root = render();`
`    expect(root.find('.Home').html()).toContain('Godzilla of browsers');`
`  });`
which should work but it gives me following error.
screen shot 2017-11-10 at 8 20 55 am

File: tests/unit/amo/containers/TestHome.js [hunk 3]
```diff
@@ -168,6 +168,11 @@ describe(__filename, () => {
     });
   });
 
+  it('renders a span with comment for IT', () => {
+    const root = render();
+    expect(root.find({ dangerouslySetInnerHTML: { __html: '<!-- Godzilla of browsers -->' } })).toHaveLength(1);
```
[the remark this answers is not in the document]
Comment (tsl143) [reply tests/unit/amo/containers/TestHome.js]: done

**Comment by willdurand** [comment]
@tsl143 I have pushed a minor tweak to your branch as I was testing it locally :)

**Comment by willdurand** [comment]
Thanks @tsl143!

**Pull request merged**